lib/swagger/swaggerTypes.ts

```diff
@@ -373,7 +373,7 @@ export interface SwaggerSpec {
   definitions?: { [definitionsName: string]: Schema };
   parameters?: { [parameterName: string]: BodyParameter | QueryParameter | FormDataParameter };
   responses?: { [responseName: string]: Response };
-  security?: Security[];
```

Comment:
This change alignes with standard swagger schema & autorest authentication schema. @raych1 pls help review